soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
+     Text{"Portrait of Oren", "Portrait d'Orlène", "Retrato de Oren"},
+     Text{"Letter to King Zora", "Lettre au roi Zora", "Carta al Rey Zora"}};
   trickNameTable[GI_FROG] = {
      Text{"Don Gero", "Don Gero", "Don Gero"},
-     Text{"Eyedrop Frog", "Grenouille-qui-louche", "Globo Ocular de Rana"},
-     Text{"Frog", "Crapaud", "Rana"}};
+     Text{"Hot-Footed Frog", "Grenouille à pieds chauds", "Rana de patas calientes"},
+     Text{"Lost Swordsmith", "Forgeron perdu", "Espadachín perdido"}};
   trickNameTable[GI_EYEDROPS] = {
-     Text{"Biggoron's Eyedrops", "Gouttes de Grogoron", "Gotas de Biggoron"},
-     Text{"Hyrule's Finest Eyedrops", "Eau du Lac Hylia", "Gotas oculares"},
+     Text{"Moon's Tear", "Larme de Lune", "Lágrima de Luna"},
+     Text{"Engine Grease", "Graisse moteur", "Grasa del motor"},
      Text{"Zora Perfume", "Parfum Zora", "Perfume Zora"}};
   trickNameTable[GI_CLAIM_CHECK] = {
-     Text{"Clay Check", "Certificat Grogoron", "Comprobante de Reclamación"},
+     Text{"Ancient Tablet", "Stèle ancienne", "Litografía arcana"},
      Text{"Sheikah Slate", "Tablette Sheikah", "Piedra Sheikah"},
      Text{"Cyclone Slate", "Ardoise des tornades", "Pizarra de los Torbellinos"}};
 
   trickNameTable[GI_SKULL_TOKEN] = {
-     Text{"Skulltula Token", "Bon de Skulltula dorée", "Símbolo de Skulltula"},
-     Text{"Golden Skulltula Spirit", "Pièce de Skulltula dorée", "Tóken de Skulltula Dorada"},
-     Text{"Gold Walltula Token", "Jeton de Walltula dorée", "Skulltula dorada"}};
+     Text{"Maiamai", "Ti'gorneau", "Maimai"},
+     Text{"Gratitude Crystal", "Cristal de gratitude", "Gema de gratitud"},
+     Text{"Korok Seed", "Noix korogu", "Semilla de kolog"}};
 
   trickNameTable[0x80] = {
      Text{"Progressive Grappling Hook", "Lance-chaîne (prog.)", "Garra progresiva"},
      Text{"Progressive Clawshot", "Grappin-griffe (prog.)", "Zarpa progresiva"},
      Text{"Progressive Gripshot", "Grappince (prog.)", "Enganchador progresivo"}};
   trickNameTable[0x81] = {
-     Text{"Progressive Glove", "Gant de puissance (prog.)", "Guanteletes progresivos"},
-     Text{"Progressive Power Bracelet", "Bracelet de force (prog.)", "Brasaletes progresivos"},
-     Text{"Progressive Magic Bracelet", "Bracelet magique (prog.)", "Manoplas progresivas"}};
+     Text{"Power Glove", "Gant de puissance (prog.)", "Guanteletes progresivos"},
+     Text{"Power Bracelet", "Bracelet de force (prog.)", "Brasaletes progresivos"},
+     Text{"Magic Bracelet", "Bracelet magique (prog.)", "Manoplas progresivas"}};
   trickNameTable[0x82] = {
-     Text{"Progressive Bomb Capacity", "Capacité de bombes (prog.)", "Mayor capacidad de bombas"},
-     Text{"Progressive Bomb Pack", "Paquet de bombes (prog.)", "Zurrón de bombas progresivo"},
-     Text{"Progressive Bomb Box", "Boîte à bombes (prog.)", "Bolsa de bombas progresiva"}};
+     Text{"Progressive Blast Mask", "Masque d'Explosion (prog.)", "Máscara explosiva progresiva"},
```

Comment:
That one was forcing the player to pay better attention, feels better to have the old one